PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+*Title* : *One line description*
+
+>Title can be a one or two words to describe the subsystem or the aspect
+ this PR applies to. For example: Build, Docs, HTTP, Upstream, SSl, ...
```

Comment:
s/SSl/TLS

Reply:
This was mentioned before but like Linux kernel, title should be all lower case. Please update the things above and mention this, with a few examples.